### DISCUSSION TITLE
SQL Server 64 bit

### DISCUSSION THREAD

**Discussion by MuhammadAliYousaf** [utmapp/UTM#3652] [UTM Help (macOS)]
Does anyone have success in installing SQL Server 64 bit?

**Comment by HRCevo** [comment]
> Does anyone have success in installing SQL Server 64 bit?

Bro, Did you success installing? Could you share that?

I have try any way as i know but still unable to install the SQL server 2019.

Thank you a lot.

**Reply by HRCevo** [reply]
Error details:
§ Error installing SQL Server Database Engine Services Instance Features
The SQL Server feature 'SQL_Engine_Core_Inst' is not in a supported state for repair, as it was never successfully configured. Only features from successful installations can be repaired. To continue, remove the specified SQL Server feature.
Error code: 0x84B40002

**Comment by MuhammadAliYousaf** [comment]
No success till yet, do let me know if you can get it installed.

**Comment by volvesgit** [comment]
Hey guys, even me i got the same problem. Has someone found how to solve this please? Thanks.

**Comment by mariapatulea** [comment]
I also have this issue, has there been a solution for it?

**Reply by volvesgit** [reply]
i got thee solution if you want

**Reply by HRCevo** [reply]
Could you share that?

**Reply by adrianguanipa** [reply]
Yes please, share the solution here, Thank you.
